update_registry.py

```diff
@@ -0,0 +1,124 @@
+import os
+import yaml
+import pytest
+from network_wrangler import ProjectCard
+import pandas as pd
+
+card_dir = os.path.join(".", "projects")
+filename_registry = "registry.csv"
```

Comment:
have "good defaults" written in ALLCAPS which the function can override.

example:

```python
CONFIG_FILENAME = "registry_config.yml"

def update_registry(registry_config:str = CONFIG_FILENAME...

```